### ISSUE TITLE
release before dplyr 1.0.8

### ISSUE THREAD

**Issue by romainfrancois** [egnha/gestalt#28]
We're about to release dplyr 1.0.8 and this package was identified as broken by our reverse dependency checks: 

````
# gestalt

<details>

* Version: 0.1.8
* GitHub: https://github.com/egnha/gestalt
* Source code: https://github.com/cran/gestalt
* Date/Publication: 2019-06-27 08:20:03 UTC
* Number of recursive dependencies: 47

Run `cloud_details(, "gestalt")` for more info

</details>

## Newly broken

*   checking tests ... ERROR
    ```
      Running ‘testthat.R’
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ Failed tests ════════════════════════════════════════════════════════════════
      ── Error (test-functions.R:65:3): body can be a closure ────────────────────────
      Error: Body must be an expression or closure
      Backtrace:
          ▆
       1. ├─testthat::expect_equal(foo, fn(x ~ function(y) x + y)) at test-functions.R:65:2
       2. │ └─testthat::quasi_label(enquo(expected), expected.label, arg = "expected")
       3. │   └─rlang::eval_bare(expr, quo_get_env(quo))
       4. └─gestalt::fn(x ~ function(y) x + y)
       5.   └─gestalt make_fn(fun$args, fun$body, ..env)
       6.     └─is_expression(body) %because% ...
      
      [ FAIL 1 | WARN 0 | SKIP 0 | PASS 1040 ]
      Error: Test failures
      Execution halted
    ```

````

The good news is that the dev version seems to fix it.

**Comment by egnha** [comment]
@romainfrancois, thanks for bringing this to my attention. The cause is an upcoming change in rlang's upcoming API. I will submit a fix (current master) to CRAN when rlang 1.0 lands.

**Comment by egnha** [comment]
Fixed in 0.1.9.
